### DISCUSSION TITLE
Exit code 1 help!

### DISCUSSION THREAD

**Discussion by cosmicowl15** [aclap-dev/video-downloadhelper#432] [Help and Q&A] [answered]
### Your question

Hi, I am trying to download a video[ here ](https://seriesflix.fit/episodio/suenos-de-libertad-1x2/) and keep getting this error code:

Exit code: 1

moz-extension://24b47ac5-2282-41af-bdbc-12aade5a0d9c/background/main.js:1
receive@moz-extension://24b47ac5-2282-41af-bdbc-12aade5a0d9c/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://24b47ac5-2282-41af-bdbc-12aade5a0d9c/background/main.js:1:42626

I have tried downloading the same video from [here](https://www.seriesonline.tax/ver-series/sueanos-de-libertad-temporada-1-capitulo-4-370818.html) as well, I get the same error. I've downloaded from this site before, but from a different server, so I wonder if that's what is causing the error? 

Browser is Firefox, OS is Windows 11. Any help is appreciated! TIA

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-CA
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files (x86)\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "i686",
    "node": "10"
  },
  "home": "C:\\Users\\tabat",
  "converterBinary": "C:\\Program Files (x86)\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by marcusguita** [comment]
I tested it here, and for this link, everything is working fine.

**Reply by GrampaWildWilly** [reply]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 125.0.2 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

I visited both your links & at each one, VDH had no problem downloading the videos.  I didn't sit & watch them, just sampled them at intervals to make sure they had both video & audio, which they did.  In both cases, I selected the first variant listed on the VDH menu.   Both sites provided decent download speeds.

**Reply by cosmicowl15** [reply, marked as answer]
Thank you both, I kept trying for quite a while and it never worked out for me. I ended up solving the problem simply by switching to an incognito window.
